libs/execution_context/ebpf_program.c

```diff
@@ -671,6 +671,8 @@ _ebpf_program_load_byte_code(
     // failing.
     ubpf_toggle_bounds_check(program->code_or_vm.vm, false);
 
+    ubpf_set_error_print(program->code_or_vm.vm, ebpf_log_function);
```

Comment:
oh so was there a bug? Shouldnt the UM code invoke this function in EC as well?

Reply:
Because of this:
https://github.com/microsoft/ebpf-for-windows/blob/980646a793c3a18e5175b3a7b012e88d30f4622a/libs/ubpf/kernel/ubpf_kernel.c#L31

uBPF prints messages via fprintf by default.